src/framework/lib/genesisframework/task.rb

```diff
@@ -81,7 +81,7 @@ def run_cmd *cmd, stdin_data: '', return_both_streams: false, return_merged_stre
           end
 
           if status.exitstatus != 0
-            log("Run Command failed for '%s' with exit status '%d' and output: %s" % [cmd.to_s, status.exitstatus, output.to_s])
+            log("Run Command failed for '%s' with exit status '%d' and output: %s", 'CRITICAL' % [cmd.to_s, status.exitstatus, output.to_s])
```

Comment:
I think the Ruby linter should complain about this. Seems you are doing % interpolation on the 'CRITICAL' string. You should add proper parens to make this semantically correct